IFC4x3/Templates/Product Shape/Product Geometric Representation/Surface Geometry/Surface Sectioned Geometry/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Surface Sectioned Geometry" UniqueId="c62dfbec-49c6-41e3-836e-d494170ec2c8" Type="IfcElement">
+	<Rules>
+		<DocModelRuleAttribute Name="Representation">
+			<Rules>
+				<DocModelRuleEntity Name="IfcProductDefinitionShape">
+					<Rules>
+						<DocModelRuleAttribute Name="Representations">
+							<Rules>
+								<DocModelRuleEntity Name="IfcShapeRepresentation">
+									<Rules>
+										<DocModelRuleAttribute Name="ContextOfItems">
+											<Rules>
+												<DocModelRuleEntity Name="IfcGeometricRepresentationContext" />
+											</Rules>
+										</DocModelRuleAttribute>
+										<DocModelRuleAttribute CardinalityMin="1" CardinalityMax="1" Name="RepresentationIdentifier" Identification="Identifier">
+											<Rules>
+												<DocModelRuleEntity Name="IfcLabel">
+													<Rules>
+														<DocModelRuleEntity xsi:nil="true" href="IfcLabel_212pjtt2TCSe6IuxauYKd"/>
+													</Rules>
+												</DocModelRuleEntity>
```

Comment:
Found the Issue Double bracketing should be fine now @larswik
```suggestion
												<DocModelRuleEntity xsi:nil="true" href="IfcLabel_212pjtt2TCSe6IuxauYKd"/>>
```

Reply:
![image](https://user-images.githubusercontent.com/74732183/140729042-9df6e814-f556-4de7-8d81-c0631180268d.png)